app/feature/feature-help-center/src/main/kotlin/com/hedvig/android/feature/help/center/di/HelpCenterModule.kt

```diff
@@ -1,13 +1,19 @@
 package com.hedvig.android.feature.help.center.di
 
+import com.apollographql.apollo3.ApolloClient
 import com.hedvig.android.data.travelcertificate.CheckTravelCertificateDestinationAvailabilityUseCase
 import com.hedvig.android.feature.help.center.HelpCenterViewModel
 import com.hedvig.android.feature.help.center.data.GetCommonClaimsUseCase
+import com.hedvig.android.feature.help.center.data.GetMemberActionsUseCase
+import com.hedvig.android.feature.help.center.data.GetMemberActionsUseCaseImpl
 import com.hedvig.android.feature.help.center.data.GetQuickLinksUseCase
+import com.hedvig.android.featureflags.FeatureManager
 import org.koin.androidx.viewmodel.dsl.viewModel
 import org.koin.dsl.module
 
 val helpCenterModule = module {
+
+  // todo remove from here
```

Comment:
Perfect, this will then mean the HelpCenter presenter can just use this new use case instead right? Will there be a follow up commit for that in this PR?

Reply:
Yep, but this use case it's not finished yet -you just said to open PR with what I have for shimmer, so I did, and this unfinished useCase just tagged along:) It's not used anywhere yet, so I think this could be merged after I fix the shimmer thing, and I will continue to work on the useCase usagefor a separate PR.